calcom/cal.com
# Fixing app details disconnect button and number of apps

## What does this PR do?

Trying to fix #4269 and #4257

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

Go to an installed app details page and see the Disconnect button to remove it. Also, trying to fix number of apps in categories in v2 where it should show more apps under certain categories.
